docs/releases/1.8.txt

```diff
@@ -1343,8 +1343,12 @@ the ``url`` that references :func:`django.contrib.sitemaps.views.sitemap`::
 
     from django.contrib.sitemaps.views import sitemap
 
-    url(r'^sitemap\.xml$', sitemap, {'sitemaps': sitemaps},
-        name='django.contrib.sitemaps.views.sitemap')
+    url(
+        r'^sitemap\.xml$',
+        sitemap,
+        {'sitemaps': sitemaps},
+        name='django.contrib.sitemaps.views.sitemap',
+    )
```

Comment:
I'm fine keeping the original format. I wouldn't worry about reformatting any release notes anyway.